jest.config.js

```diff
@@ -23,40 +23,52 @@ module.exports = {
       ...projectDefault,
       displayName: 'HTTP-SERVER',
       testMatch: ['<rootDir>/packages/http-server/src/**/__tests__/*.*.ts'],
-      globals: {
-        'ts-jest': {
-          tsconfig: '<rootDir>/packages/tsconfig.test.json',
-        },
+      transform: {
```

Comment:
when attempting to run tests I was getting 
```
ts-jest[ts-jest-transformer] (WARN) Define `ts-jest` config under `globals` is deprecated. Please do
transform: {
    <transform_regex>: ['ts-jest', { /* ts-jest config goes here in Jest */ }],
},
```
so I moved the global settings for ts-jest into the transform section as recommended